app/views/admin/admin_users/show.html.erb

```diff
@@ -5,6 +5,7 @@
 
   <!--subnav-->
   <ul class="navigation actions" role="navigation">
```

Comment:
I think we also wanted to add `<p class="note">To fix common errors with this user's Subscriptions and Stats pages, and to reindex their works and bookmarks, choose "Troubleshoot Account."<p>` on this page, probably above the <h3> for the form.